src/query.rs

```diff
@@ -135,6 +135,7 @@ impl<T: Component> Query for &'_ T {
 }
 
 unsafe impl<T> QueryShared for &'_ T {}
+unsafe impl QueryShared for Entity {}
```

Comment:
style: group trait implementations under either the definition (and impl block, where applicable) of the type, or under the definition of the trait (as done for `impl Query for Entity`).

Reply:
👍